edge-functions/ab-testing-statsig/README.md

```diff
@@ -0,0 +1,41 @@
+# A/B testing with Statsig
+
+This example shows how to do A/B testing using statsig
+
+## Demo
+
+https://edge-ab-testing-statsig.vercel.app
+
+## How to Use
+
+You can choose from one of the following two methods to use this repository:
+
+### One-Click Deploy
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/ab-testing-statsig&project-name=ab-testing-statsig&repository-name=ab-testing-statsig)
+
+### Clone and Deploy
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npx create-next-app --example https://github.com/vercel/examples/tree/main/edge-functions/ab-testing-statsig
+# or
+yarn create next-app --example https://github.com/vercel/examples/tree/main/edge-functions/ab-testing-statsig
+```
+
+Next, run Next.js in development mode:
```

Comment:
Before talking running in in dev, we should mention how to get the env variable to connect statsig